kalitectl.py

```diff
@@ -80,7 +80,13 @@
 # Necessary for loading default settings from kalite
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "kalite.settings")
 
-KALITE_HOME = os.path.join(os.path.expanduser("~"), ".kalite")
+from django.conf import settings
```

Comment:
@66eli77 unless you have an alternative solution, I'm going to make a PR that makes it possible to use an env. variable for this. You just can't import django settings here.